tools/packaging/kernel/build-kernel.sh

```diff
@@ -88,16 +88,17 @@ Commands:
 
 Options:
 
-	-c <path>   : Path to config file to build a the kernel.
-	-d          : Enable bash debug.
-	-e          : Enable experimental kernel.
-	-f          : Enable force generate config when setup.
-	-g <vendor> : GPU vendor, intel or nvidia.
-	-h          : Display this help.
-	-k <path>   : Path to kernel to build.
-	-p <path>   : Path to a directory with patches to apply to kernel.
-	-t          : Hypervisor_target.
-	-v          : Kernel version to use if kernel path not provided.
+	-a <arch>   	: Arch target to build the kernel.
```

Comment:
Adding https://github.com/kata-containers/kata-containers/blob/2.0-dev/tools/packaging/kernel/build-kernel.sh#L110-L113 as option candidate will be more helpful, like `-g <vendor>` options.

Reply:
ok, thanks. I recommit a patch later.